tests/dummy/app/routes/nodes/detail/draft-registrations/index.js

```diff
@@ -8,6 +8,8 @@ export default Ember.Route.extend({
     },
     setupController(controller, model) {
         this._super(controller, model);
+        var metaschemas = this.store.findAll('metaschema');
+        controller.set('metaschemas', metaschemas);
```

Comment:
Also: aren't metaschemas the possible kinds of registrations? (the question forms) 

It seems like what you really want is a separate or nested route: one page to show your existing registrations, and a separate but related one to create a new registration. That nested route could render as part of the page, but with its own model hook.

Not sure if it's a fit for your use case, but a useful option to have in your pocket.